src/lib/holocene/input/input.svelte

```diff
@@ -41,7 +42,9 @@
 
 <div class={$$props.class}>
   {#if label}
-    <label for={id}>{label}</label>
+    <label for={id}
+      >{label}{#if required}*{/if}</label
```

Comment:
We should do the `*` with CSS. Maybe something like `class:required` and then:

```css
.required {
  @apply after:content-['*']
}
```

Reply:
I thought I had heard that's not great for screen reader users, but if have the correct attributes on the input I guess we should be ok. I'll do a little more research on the best way to handle this

Reply:
Maybe we abstract in the label if that's the case, but I would assume that the required property on the input field would inform the screen reader.